# 移動物体の中心・バウンディングボックス・速度

int32 id

geometry_msgs/Pose center
geometry_msgs/Vector3 velocity
geometry_msgs/Vector3 bounding_box
